### PR TITLE
Clean up boilerplate and add state reducers.

### PR DESCRIPTION
**Pull request by travisbaratcart**

### CODE REVIEW

File: src/reducers/index.ts [hunk 1]
```diff
@@ -1,10 +1,73 @@
...
-  todos
-});
+export const reducer: Reducer<RootState> = (state: RootState, action: any) => {
+  switch (action.type) {
+    case actions.RECEIVE_SURVEYS:
+      return Object.assign({}, state, {
+        surveys: action.surveys
+      });
+
+    case actions.CHOOSE_SURVEY:
```
Comment (evels) [inline src/reducers/index.ts:41]: need choose question

File: src/reducers/index.ts [hunk 2]
```diff
@@ -1,10 +1,73 @@
...
+          if (answer.questionId === action.questionId) {
+            return {
+              questionId: answer.questionId,
+              optionId: action.optionId
+            };
+          } else {
+            return answer
+          }
+        })
+      });
```
Comment (evels) [inline src/reducers/index.ts:68]: need submit answers

Comment (travisbaratcart) [reply src/reducers/index.ts:68]: holding off on this until implementation is solidified

File: src/reducers/index.ts (hunk 1 above)
Comment (travisbaratcart) [reply src/reducers/index.ts:41]: added

**Comment by evels** [comment]
lgtm

**Pull request merged**